apps/api/src/routers/guest.py

```diff
@@ -72,6 +76,6 @@ async def verify_guest(
     log.info("%s authenticated as guest.", email)
     guest = guest_auth.acquire_guest_identity(email)
 
-    res = RedirectResponse("/portal", status_code=status.HTTP_303_SEE_OTHER)
+    res = RedirectResponse(return_to, status_code=status.HTTP_303_SEE_OTHER)
```

Comment:
Note: for security purposes, we'll want to confirm this destination is on our own server. The simple check is making sure it starts with `/`.